### DISCUSSION TITLE
What font is this

### DISCUSSION THREAD

**Discussion by andrewbernard** [wezterm/wezterm#2811] [Q&A] [answered]
What is this superb cursive Italic font as shown in the illustration of wezterm in the documentation?

![image](https://user-images.githubusercontent.com/4009723/204502580-29fdfe53-b3b3-4991-ac87-fe7e7020d9fd.png)

**Comment by yutkat** [comment, marked as answer]
Operator Mono SSm?
https://www.typography.com/fonts/operator/styles/screensmart

**Comment by andrewbernard** [comment]
Thank you.  A beautiful font family.
